services/brig/src/Brig/IO/Intra.hs

```diff
@@ -483,6 +484,7 @@ createSelfConv u = do
         . zUser u
         . expect2xx
 
+-- | Calls 'Galley.API.createConnectConversationH'.
```

Comment:
Thanks for this. these things having slightly different naming conventions is realllyyy irritating me though :D